frontend/src/components/Table/index.tsx

```diff
@@ -0,0 +1,21 @@
+import Table from './Table';
+export default Table;
+
+export * from './TableHeader';
+export * from './TableHeaderColumn';
+export * from './TableHeaderColumnEmpty';
+export * from './TableBody';
+export * from './TableBodyRow';
+export * from './TableBodyRowData';
+
+export const people = [
+  {
+    name: 'Jane Cooper',
+    title: 'Regional Paradigm Technician',
+    department: 'Optimization',
+    role: 'Admin',
+    email: 'jane.cooper@example.com',
+    image:
+      'https://images.unsplash.com/photo-1494790108377-be9c29b29330?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=4&w=256&h=256&q=60',
+  },
+];
```

Comment:
do you still need this file? i could not find where you called and used this file.

Reply:
yes po, the use is present on later PRS,

one of the PR include the refactor for the imports

![image](https://user-images.githubusercontent.com/95029803/145946294-2d7a08d0-089e-40da-b56a-91e9f98d4a50.png)
![image](https://user-images.githubusercontent.com/95029803/145946312-2759c231-f15c-4485-8f13-947c48ad6f23.png)

Reply:
PR 15 was the pr where i mentioned i refactored the imports

Reply:
ahh okay then this is resolved :bow: